src/content/blog/2024-09-29-fighting-the-invisible-Battle-my-story-of-health-struggles.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: "Fighting the Invisible Battle: My Story of Health Struggles"
+description: "Hey friends, I wish you a healthy life and hope you are doing great physically and mentally, please take care of your health."
+date: 2024-09-29T09:38:23.854Z
+preview: ../images/health/wellness.png
+coverSVG: ../images/svg/undraw/drew-beamer-xU5Mqq0Chck-unsplash.svg
+socialImage: ../images/health/wellness.png
+draft: false
+tags: ['health','family','community']
+categories: ['community']
+publishDate: 2024-10-08T20:48:29.436Z
+author: Santosh Yadav
+---
+
+import doubt from '../../images/health/doubt.jpg';
+import myself from '../../images/health/healthy.jpg';
+import wellness from '../../images/health/wellness.png';
+import speaking from '../../images/undraw/public-speaking.jpeg';
+import excuse from '../../images/health/excuse.jpg';
+
+import BlogImage from '../../components/blogimage.astro';
+
+Hey friends, if you are here, I wish you a healthy life and hope you are doing great physically and mentally. And please take care of your health. I learned it the hard way. Though I always preach about work-life balance, I ended up screwing up my health. 
+
+I will share my journey, how I fought an invisible battle, and where I am currently.
+
+# What happened 
+
+In April 2024, my wife and I went for a full-body checkup, as we had hardly done it since we moved to Germany. We got our reports after a week, and my doctor gave me the bad news, I had symptoms of diabetes, and my blood sugar was 270 in the report. She was surprised at how I could do my day-to-day work as I should have mainly been sleeping; she was not wrong; that was what I was doing, and my doctor mentioned she could write me a note to take some days off.
+
+## Struggle with health
```

Comment:
Maybe "Impact on Mental Health" is better?

Reply:
good suggestion